msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Add, update, and remove pre-approvals for RSC enabled applications in Teams
+description: In this article, learn more about creating, managing, and deleting pre-approvals for RSC enabled applications in MS Teams
+ms.localizationpriority: medium
+author: MSFTRickyCastaneda
+ms.author: rcastaneda
+ms.topic: Conceptual
+ms.date: 08/29/2023
+---
+
+In this article:
+1. [What is preapproval of RSC permissions](#what-is-preapproval-of-rsc-permissions)
+1. [Setting up PowerShell to manage preapprovals](#setting-up-powershell-to-manage-preapprovals)
+1. [Enable pre-approval for RSC permissions](#enable-preapproval-for-rsc-permissions)
+1. [Create a preapproval](#create-a-preapproval)
+      * [With a sensitivity label](#create-a-preapproval-with-a-sensitivity-label)
+      * [Without a sensitivity label](#create-a-preapproval-without-a-sensitivity-label)
+1. [Modify a preapproval](#modify-a-preapproval)
+1. [Delete a preapproval](#delete-a-preapproval)
+
+# What is preapproval of RSC permissions?
+
+Preapproval of RSC permissions allows admins to approve resource-specific consent (RSC) on an app-by-app basis in Microsoft Teams. Previously, admins had the ability to toggle RSC on or off in their tenant. Admins had to approve or block all RSC enabled apps with no granularity to declare which apps they wanted to allow. Through pre-approved RSC, admins can choose which RSC-enabled apps they allow their tenant users to install in Teams without having to allow or block all apps that utilize RSC permissions.
+
+   > [!NOTE]
+   > RSC Pre-approval is currently in public preview, features and operating procedures are subject to change.
+
+# Setting up PowerShell to manage preapprovals
+
+Preapproval of RSC permissions is managed through Microsoft Teams PowerShell. You can learn more about managing Microsoft Teams with PowerShell [here](/manage-teams-with-microsoft-teams-powershell)
```

Comment:
>(/manage-teams-with-microsoft-teams-powershell) [](http://example.com/codeflow?start=150&length=47)

This is incorrect. This takes us to the Teams powershell. We should link to graph powershell